packages/bubbles-map/static/manifest.dev.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Nice use for the dev manifest!

Could you please also add a sample dataset here? See this as an example:

https://github.com/jesuino/dashbuilder-components/blob/using-api/packages/hello-component/static/manifest.dev.json#L8